### PR TITLE
Feature/ds 189

### PR DESCRIPTION
**Pull request by MichalPaszowski**

### CODE REVIEW

File: packages/react-components/src/components/dropdown/helpers.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+): null | number {
+  const focusableItem: IDropdownListItem | undefined = items.find(
+    (item) => !item.isDisabled
+  );
+
+  if (!focusableItem) {
+    return null;
+  }
+
+  return focusableItem.itemId;
```
Comment (MichalPaszowski) [inline packages/react-components/src/components/dropdown/helpers.ts:15]: here eslint says that I return any type

File: packages/react-components/src/components/dropdown/helpers.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    (item: IDropdownListItem) =>
+      !item.isDisabled && item.itemId !== focusedItemId
+  );
+
+  if (keyCode === KeyCodes.arrowUp) {
+    activeItems = activeItems.reverse();
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  return activeItems[0];
```
Comment (MichalPaszowski) [inline packages/react-components/src/components/dropdown/helpers.ts:49]: here eslint says that I return any type

**Comment by skrabkab-design** [comment]
1. Let's remove the tick from dropdown component, so that the difference between tooltip & dropdown is more obvious.
2. Let's remove "examples" from navigation 
3. Let's add placeholder to Dropdown, to better reflect paddings & patterns @skrabkab-design 
4. Selectable vs. Non-selectable - think about better name for this variant @skrabkab-design
5. Better reflect menu example - design @skrabkab-design 
6. Please add list component - without positioning and trigger (html & css example)
7. Let's think about improvements in Docs part, so that preview is not that big, and easier to scroll through.

**Comment by skrabkab-design** [comment]
> 3. Let's add placeholder to Dropdown, to better reflect paddings & patterns @skrabkab-design
https://www.figma.com/file/2pFu80PXO5A2tfyrAGnx91/Product-Components?node-id=5232%3A39682
> 4. Selectable vs. Non-selectable - think about better name for this variant @skrabkab-design
Let's name it this way:
Selectable -> [Dropdown List ](https://www.figma.com/file/2pFu80PXO5A2tfyrAGnx91/Product-Components?node-id=5232%3A39695)
Non-selectable -> [Action Menu](https://www.figma.com/file/2pFu80PXO5A2tfyrAGnx91/Product-Components?node-id=5232%3A39772)
> 5. Better reflect menu example - design @skrabkab-design
https://www.figma.com/file/2pFu80PXO5A2tfyrAGnx91/Product-Components?node-id=5232%3A39772

@MichalPaszowski ☝️ All updated :)

**Pull request closed without merging**